std_msgs/Header header
string command
time start_time
time stop_time
